<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainTicketsWindow</class>
 <widget class="QMainWindow" name="MainTicketsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sistema de GestÃ£o de Tickets</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #f8f9fa;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="sidebar_widget">
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #2c3e50;</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="logo_label">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>80</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: #34495e; color: white; font-size: 18px; font-weight: bold;</string>
         </property>
         <property name="text">
          <string>ðŸŽ« Sistema de Tickets</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="dashboard_btn">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    background-color: transparent;
    color: white;
    border: none;
    text-align: left;
    padding: 15px 20px;
    font-size: 14px;
}
QPushButton:hover {
    background-color: #34495e;
}
QPushButton:checked {
    background-color: #3498db;
}</string>
         </property>
         <property name="text">
          <string>ðŸ“Š Dashboard</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="tickets_btn">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    background-color: transparent;
    color: white;
    border: none;
    text-align: left;
    padding: 15px 20px;
    font-size: 14px;
}
QPushButton:hover {
    background-color: #34495e;
}
QPushButton:checked {
    background-color: #3498db;
}</string>
         </property>
         <property name="text">
          <string>ðŸŽ« Meus Tickets</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="new_ticket_btn">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    background-color: transparent;
    color: white;
    border: none;
    text-align: left;
    padding: 15px 20px;
    font-size: 14px;
}
QPushButton:hover {
    background-color: #34495e;
}
QPushButton:checked {
    background-color: #3498db;
}</string>
         </property>
         <property name="text">
          <string>âž• Novo Ticket</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="users_btn">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    background-color: transparent;
    color: white;
    border: none;
    text-align: left;
    padding: 15px 20px;
    font-size: 14px;
}
QPushButton:hover {
    background-color: #34495e;
}
QPushButton:checked {
    background-color: #3498db;
}</string>
         </property>
         <property name="text">
          <string>ðŸ‘¥ UsuÃ¡rios</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdSet="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="logout_btn">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    background-color: #e74c3c;
    color: white;
    border: none;
    text-align: center;
    padding: 15px 20px;
    font-size: 14px;
    font-weight: bold;
}
QPushButton:hover {
    background-color: #c0392b;
}</string>
         </property>
         <property name="text">
          <string>ðŸšª Sair</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="styleSheet">
       <string notr="true">background-color: white;</string>
      </property>
      <widget class="QWidget" name="dashboard_page">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>20</number>
        </property>
        <property name="leftMargin">
         <number>30</number>
        </property>
        <property name="topMargin">
         <number>30</number>
        </property>
        <property name="rightMargin">
         <number>30</number>
        </property>
        <property name="bottomMargin">
         <number>30</number>
        </property>
        <item>
         <widget class="QLabel" name="dashboard_title">
          <property name="text">
           <string>Dashboard</string>
          </property>
          <property name="font">
           <font>
            <pointsize>24</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: #2c3e50;</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QFrame" name="stats_frame_1">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>120</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: #3498db; border-radius: 10px; color: white;</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QLabel" name="tickets_abertos_label">
               <property name="text">
                <string>0</string>
               </property>
               <property name="font">
                <font>
                 <pointsize>32</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">color: white;</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="tickets_abertos_text">
               <property name="text">
                <string>Tickets Abertos</string>
               </property>
               <property name="styleSheet">
                <string notr="true">color: white; font-size: 14px;</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="stats_frame_2">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>120</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: #e67e22; border-radius: 10px; color: white;</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QLabel" name="tickets_pausados_label">
               <property name="text">
                <string>0</string>
               </property>
               <property name="font">
                <font>
                 <pointsize>32</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">color: white;</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="tickets_pausados_text">
               <property name="text">
                <string>Tickets Pausados</string>
               </property>
               <property name="styleSheet">
                <string notr="true">color: white; font-size: 14px;</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="stats_frame_3">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>120</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: #27ae60; border-radius: 10px; color: white;</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <widget class="QLabel" name="tickets_fechados_label">
               <property name="text">
                <string>0</string>
               </property>
               <property name="font">
                <font>
                 <pointsize>32</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">color: white;</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="tickets_fechados_text">
               <property name="text">
                <string>Tickets Fechados</string>
               </property>
               <property name="styleSheet">
                <string notr="true">color: white; font-size: 14px;</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="recent_tickets_label">
          <property name="text">
           <string>Tickets Recentes</string>
          </property>
          <property name="font">
           <font>
            <pointsize>18</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: #2c3e50;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="recent_tickets_table">
          <property name="styleSheet">
           <string notr="true">QTableWidget {
    border: 1px solid #bdc3c7;
    border-radius: 5px;
    background-color: white;
}
QTableWidget::item {
    padding: 8px;
}
QHeaderView::section {
    background-color: #ecf0f1;
    padding: 8px;
    border: none;
    border-right: 1px solid #bdc3c7;
    border-bottom: 1px solid #bdc3c7;
}</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tickets_page">
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <property name="spacing">
         <number>20</number>
        </property>
        <property name="leftMargin">
         <number>30</number>
        </property>
        <property name="topMargin">
         <number>30</number>
        </property>
        <property name="rightMargin">
         <number>30</number>
        </property>
        <property name="bottomMargin">
         <number>30</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="tickets_title">
            <property name="text">
             <string>Meus Tickets</string>
            </property>
            <property name="font">
             <font>
              <pointsize>24</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: #2c3e50;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="status_filter">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>35</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QComboBox {
    border: 2px solid #bdc3c7;
    border-radius: 5px;
    padding: 5px;
    font-size: 14px;
}</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdSet="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="tickets_table">
          <property name="styleSheet">
           <string notr="true">QTableWidget {
    border: 1px solid #bdc3c7;
    border-radius: 5px;
    background-color: white;
}
QTableWidget::item {
    padding: 8px;
}
QHeaderView::section {
    background-color: #ecf0f1;
    padding: 8px;
    border: none;
    border-right: 1px solid #bdc3c7;
    border-bottom: 1px solid #bdc3c7;
}</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="new_ticket_page">
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <property name="spacing">
         <number>20</number>
        </property>
        <property name="leftMargin">
         <number>30</number>
        </property>
        <property name="topMargin">
         <number>30</number>
        </property>
        <property name="rightMargin">
         <number>30</number>
        </property>
        <property name="bottomMargin">
         <number>30</number>
        </property>
        <item>
         <widget class="QLabel" name="new_ticket_title">
          <property name="text">
           <string>Novo Ticket</string>
          </property>
          <property name="font">
           <font>
            <pointsize>24</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: #2c3e50;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="ticket_form_frame">
          <property name="styleSheet">
           <string notr="true">background-color: white; border: 1px solid #bdc3c7; border-radius: 10px;</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <property name="spacing">
            <number>20</number>
           </property>
           <property name="leftMargin">
            <number>30</number>
           </property>
           <property name="topMargin">
            <number>30</number>
           </property>
           <property name="rightMargin">
            <number>30</number>
           </property>
           <property name="bottomMargin">
            <number>30</number>
           </property>
           <item>
            <widget class="QLabel" name="titulo_label">
             <property name="text">
              <string>TÃ­tulo:</string>
             </property>
             <property name="styleSheet">
              <string notr="true">color: #34495e; font-weight: bold; font-size: 14px;</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="titulo_input">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>40</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QLineEdit {
    border: 2px solid #bdc3c7;
    border-radius: 5px;
    padding: 8px;
    font-size: 14px;
}
QLineEdit:focus {
    border-color: #3498db;
}</string>
             </property>
             <property name="placeholderText">
              <string>Digite o tÃ­tulo do ticket</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="descricao_label">
             <property name="text">
              <string>DescriÃ§Ã£o:</string>
             </property>
             <property name="styleSheet">
              <string notr="true">color: #34495e; font-weight: bold; font-size: 14px;</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTextEdit" name="descricao_input">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>120</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QTextEdit {
    border: 2px solid #bdc3c7;
    border-radius: 5px;
    padding: 8px;
    font-size: 14px;
}
QTextEdit:focus {
    border-color: #3498db;
}</string>
             </property>
             <property name="placeholderText">
              <string>Descreva detalhadamente o problema ou solicitaÃ§Ã£o</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdSet="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="cancelar_btn">
               <property name="minimumSize">
                <size>
                 <width>120</width>
                 <height>40</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton {
    background-color: #95a5a6;
    color: white;
    border: none;
    border-radius: 5px;
    font-size: 14px;
    font-weight: bold;
}
QPushButton:hover {
    background-color: #7f8c8d;
}</string>
               </property>
               <property name="text">
                <string>Cancelar</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="criar_ticket_btn">
               <property name="minimumSize">
                <size>
                 <width>120</width>
                 <height>40</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton {
    background-color: #3498db;
    color: white;
    border: none;
    border-radius: 5px;
    font-size: 14px;
    font-weight: bold;
}
QPushButton:hover {
    background-color: #2980b9;
}</string>
               </property>
               <property name="text">
                <string>Criar Ticket</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="users_page">
       <layout class="QVBoxLayout" name="verticalLayout_9">
        <property name="spacing">
         <number>20</number>
        </property>
        <property name="leftMargin">
         <number>30</number>
        </property>
        <property name="topMargin">
         <number>30</number>
        </property>
        <property name="rightMargin">
         <number>30</number>
        </property>
        <property name="bottomMargin">
         <number>30</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="users_title">
            <property name="text">
             <string>Gerenciar UsuÃ¡rios</string>
            </property>
            <property name="font">
             <font>
              <pointsize>24</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: #2c3e50;</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdSet="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="add_user_btn">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>40</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
    background-color: #27ae60;
    color: white;
    border: none;
    border-radius: 5px;
    font-size: 14px;
    font-weight: bold;
}
QPushButton:hover {
    background-color: #229954;
}</string>
            </property>
            <property name="text">
             <string>+ Adicionar UsuÃ¡rio</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="users_table">
          <property name="styleSheet">
           <string notr="true">QTableWidget {
    border: 1px solid #bdc3c7;
    border-radius: 5px;
    background-color: white;
}
QTableWidget::item {
    padding: 8px;
}
QHeaderView::section {
    background-color: #ecf0f1;
    padding: 8px;
    border: none;
    border-right: 1px solid #bdc3c7;
    border-bottom: 1px solid #bdc3c7;
}</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
